.jenkins/pytorch/win-test-helpers/build_pytorch.bat

```diff
@@ -73,6 +73,8 @@ set PATH=%TMP_DIR_WIN%\bin;%PATH%
 :: jenkins has M40, which is 5.2
 if "%TORCH_CUDA_ARCH_LIST%" == "" set TORCH_CUDA_ARCH_LIST=5.2
 
+:: The default sccache idle timeout is 600, which is too short and leads to intermittent build errors.
+set SCCACHE_IDLE_TIMEOUT=0
```

Comment:
(Assuming the units here are seconds.) Would increasing it to 20 min help solve intermittent build issues?
And can you please post a few example, where 10 min timeout was not enough?

Reply:
I'm guessing that sometimes MSVC just takes a really long to build the files?

Reply:
More than 10 min? Wow!

Reply:
@malfet It doesn't mean building a single source file takes 10 min. It just mean that there have been 10 min without calling it. Considering the linking process of `torch_cpu.dll` takes nearly 7 min in the PR jobs, it is quite normal. As you can see in `.jenkins/pytorch/common.sh`, it is already set to 1200.

Reply:
> (Assuming the units here are seconds.) Would increasing it to 20 min help solve intermittent build issues?
> And can you please post a few example, where 10 min timeout was not enough?

Sorry, I cannot find it anymore. Before I send this PR, I just randomly skimmed the ci.pytorch.org HUD and found one occurrence of the build failure that is caused by the timeout.